crypto/src/test/java/org/hyperledger/besu/crypto/SignatureAlgorithmTypeTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *  * Copyright ConsenSys AG.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
```

Comment:
Same, the license format seems to be wrong

Reply:
Thanks. I've fixed it